ios/StripeTerminalReactNative.swift

```diff
@@ -496,12 +552,13 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
             resolve(Errors.createError(code: CommonErrorType.InvalidRequiredParameter, message: "You must provide paymentIntentId."))
             return
         }
-        guard let paymentIntent = paymentIntents[id] else {
+        
```

Comment:
I can comment on the line, but we should rename all `processPayment` and `processRefund` to `confirmPaymentIntent` and `confirmRefund`